content/02.main-text.md

```diff
@@ -21,12 +21,12 @@ The manuscript writing process we developed using the Markdown language, the Git
 
 ## Manubot contribution workflow
 
-There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats, to LaTeX-based systems for technical writing [@doi:10.1038/514127a] such as [Overleaf](https://www.overleaf.com/) and [Authorea](https://www.authorea.com/).
+There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents or similar formats and provide "what you see is what you get" (WYSIWYG) interfaces, to LaTeX-based systems for technical writing [@doi:10.1038/514127a] such as [Overleaf](https://www.overleaf.com/) and [Authorea](https://www.authorea.com/).
```

Comment:
>There are many existing collaborative writing platforms ranging from rich text editors, which support Microsoft Word documents 

I'm not sure I understand the first clause. Just because something is a rich text editor doesn't necessarily imply it supports Microsoft Word, right (e.g., Overleaf v2 which has a rich text interface but doesn't directly support Word)?

Reply:
I rephrased this paragraph to shorten some sentences and hopefully make it flow better in 97f7c32.  Did that improve the rich text / Microsoft Word description?  The intention here is to refer to Word and Google Docs, not rich text modes of more sophisticated tools.

Reply:
Yes!